shared/desktop/yarn-helper/font.js

```diff
@@ -77,6 +77,18 @@ const getSvgPaths = skipUnmatchedFile =>
  * For config options: https://github.com/sunflowerdeath/webfonts-generator
  */
 function updateIconFont(web) {
+  if (!web) {
+    // Check if fontforge is installed, required to generate the font
+    try {
+      execSync('fontforge')
+    } catch (e) {
+      if (e.message.includes('not found')) {
+        throw new Error('FontForge is required to generate the icon font. Install and try again.')
```

Comment:
you should say they need to run `yarn` and not `yarn modules` and install fontforge globally

Reply:
fontforge isn't installed by yarn though

Reply:
yeah additionally they need to install fontforge globally, they need both and its not the usual flow